src/Application/HttpModels/MandateCreate.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace MatchBot\Application\HttpModels;
+
+use MatchBot\Domain\Campaign;
+use MatchBot\Domain\Currency;
+use MatchBot\Domain\DayOfMonth;
+use MatchBot\Domain\Money;
+use MatchBot\Domain\Salesforce18Id;
+
+/**
+ * Deserializable DTO for request to create a new regular giving mandate
+ */
+readonly class MandateCreate
+{
+    public Money $amount;
+    public DayOfMonth $dayOfMonth;
+
+    /** @var Salesforce18Id<Campaign>  */
+    public Salesforce18Id $campaignId;
+
+    /**
+     * @psalm-suppress PossiblyUnusedMethod - called by Symfony Serializer
+     */
+    public function __construct(
+        // not taking any value objects as constructor params as at least with our current config or
+        // any config I've been able to find the Symfony Serializer doesn't initialise their properties.
```

Comment:
I think this is all good, but would be helpful to add the new schema & endpoint to the OpenAPI as we may need to get updates to the vendor quite quickly if they come back on a time for testing before CC

Reply:
Right - I thought I'd implement the consumer in frontend though, and there will probably be a bit of iteration on API design between the two. Do you think we should add stuff to the OpenAPI spec from the start and keep it updated with every change we merge to matchbot?

Reply:
I don't think we need to incorporate every change but we might end up backed into having to do things fairly urgently if we don't do an initial draft as we go. It's a bit hard to say until I can get hold of the vendor again, so let's just keep it in mind / chat in standup

Reply:
OK, I've pushed a couple of commits to add the create request to the api spec. I also adjusted the spec there that's already there for the returned object as used in `regular-giving/my-donation-mandates` to better match the implementation, and returned the same type of object in the new endpoint.